### PR TITLE
Make phylo trees available to system admins

### PR DESCRIPTION
**Pull request by ttung**
### Description
Regardless of group membership, system admins can see all.

#### Issue
[ch138412](https://app.clubhouse.io/genepi/story/138412/admins-cannot-interact-with-phylo-trees)

### Test plan
added test

### CODE REVIEW

File: src/backend/aspen/app/views/phylo_trees.py [hunk 1]
```diff
@@ -76,13 +76,13 @@ def phylo_trees():
...
             .filter(
                 or_(
+                    user.system_admin,
                     phylo_run_alias.group_id.in_(
                         db_session.query(Group.id)
                         .join(User)
                         .filter(User.auth0_user_id == profile["user_id"])
                         .subquery()
                     ),
-                    user.system_admin,
```
Comment (phoenixAja) [inline src/backend/aspen/app/views/phylo_trees.py:85]: wait so moving this up is what fixed it? That seems really weird to me

**Review by phoenixAja — APPROVED** [review]
awesome thanks for fixing this!!

File: src/backend/aspen/app/views/phylo_trees.py [hunk 2]
```diff
@@ -137,7 +137,12 @@ def _process_phylo_tree(
             db_session.query(PhyloTree)
             .join(PhyloRun)
             .filter(PhyloTree.entity_id == phylo_tree_id)
-            .filter(PhyloRun.group_id.in_(can_see_group_ids_trees))
+            .filter(
+                or_(
+                    user.system_admin,
+                    PhyloRun.group_id.in_(can_see_group_ids_trees),
+                )
+            )
```
Comment (ttung) [inline src/backend/aspen/app/views/phylo_trees.py:145]: no, this fixed it.

File: src/backend/aspen/app/views/phylo_trees.py (hunk 1 above)
Comment (ttung) [reply src/backend/aspen/app/views/phylo_trees.py:85]: In this case, I just wanted to group the `phylo_run_alias.group_id.in_(` statements.

**Pull request merged**